src/api/dishes/dishes.service.ts

```diff
@@ -0,0 +1,54 @@
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+export const getAllDishes = () => {
+  return prisma.dishes.findMany();
+};
+
+export const createDish = (input: any) => {
+  return prisma.dishes.create({
+    data: {
+      title: input.title,
+      description: input.description,
+      price: input.price,
+      restaurantsId_restaurant: input.restaurantsId_restaurant,
+      dishes_categoriesId_dishes_category:
+        input.dishes_categoriesId_dishes_category,
+      order_details: input.order_details,
+    },
+  });
+};
+
+export const getDishById = (id: string) => {
+  return prisma.dishes.findUnique({
+    where: {
+      id_dish: id,
+    },
+  });
+};
+
+export const updateDishById = (id: string, input: any) => {
+  return prisma.dishes.update({
+    where: {
+      id_dish: id,
+    },
+    data: {
+      title: input.title,
+      description: input.description,
+      price: input.price, //ó parceInt
```

Comment:
is necessary a parseInt or not?

Reply:
parseInt is'n necessary because input.price is a number

Reply:
ok, thx 4 clarify.